app/assets/javascripts/Measure.js

```diff
@@ -14,33 +14,29 @@ const [mNumber, mString, mBoolean, Mixed, ObjectId, mDate] = [
 
 const MeasureSchema = mongoose.Schema(
   {
-    // ID/other measure information
-    id: mString,
-    measure_id: mString,
-    hqmf_id: mString,
-    hqmf_set_id: mString,
-    hqmf_version_number: mNumber,
-    cms_id: mString,
-    title: mString,
-    description: mString,
-    type: mString,
-    category: { type: mString, default: 'Uncategorized' },
+    // A version-specific UUID for the measure
+    hqmf_id: String,
+    // A version-neutral UUID for the measure
+    hqmf_set_id: String,
+    // A Semantic Version-compliant string (e.g. "2.3.4") for the measure
+    hqmf_version_number: String,
+    // A CMS-style string (e.g. "CMS2v4") for the measure
+    cms_id: String,
+    title: String,
+    description: String,
+    // EP/EH, or "unknown"
+    type: String,
```

Comment:
comment should fully list the options.

"EH", "EP", "unknown" (or whatever these actually are)

Should this be an enum?

Reply:
These are the only options I saw in Bonnie, other than `null`, when I ran a `db.getCollection('cql_measures').distinct('type')`. I don't know that the type is (or should be) restricted to only those options, but I'd be willing to change it to be an enum if others want me to.

Reply:
Were the options `["EP", "EH", "unknown"]` or `["EP/EH", "unknown"]`?
It is not clear to me from the comment which it is - the comment is requesting that this be clarified.
If it's the latter, please put "EP/EH" in quotes.
It should be explicit to the user what the options are.

Regarding whether it should be in an enum or not, that can be something that we take a look at later. I think it should be - but i agree we should confirm what the total possible options are before we limit things.

Reply:
Ah, I'm sorry I didn't get that the first time. I'll make that more clear from the comment.

Reply:
this doesn't look like it was clarified